desktop/src/main/java/bisq/desktop/main/portfolio/openoffer/OpenOffersView.java

```diff
@@ -359,32 +387,37 @@ private void onActivateOpenOffer(OpenOffer openOffer) {
     private void onRemoveOpenOffer(OpenOfferListItem item) {
         OpenOffer openOffer = item.getOpenOffer();
         if (model.isBootstrappedOrShowPopup()) {
-            String key = (openOffer.getOffer().isBsqSwapOffer() ? "RemoveBsqSwapWarning" : "RemoveOfferWarning");
-            if (DontShowAgainLookup.showAgain(key)) {
-                String message = openOffer.getOffer().isBsqSwapOffer() ?
-                        Res.get("popup.warning.removeNoFeeOffer") :
-                        Res.get("popup.warning.removeOffer", item.getMakerFeeAsString());
-                new Popup().warning(message)
-                        .actionButtonText(Res.get("shared.removeOffer"))
-                        .onAction(() -> doRemoveOpenOffer(openOffer))
-                        .closeButtonText(Res.get("shared.dontRemoveOffer"))
-                        .dontShowAgainId(key)
-                        .show();
-            } else {
+            if (openOfferManager.safeRemovalOfOcoClone(openOffer)) {
                 doRemoveOpenOffer(openOffer);
```

Comment:
Why do we skip the popup here?

Reply:
Because there is no cost to removing a cloned offer.  In the demonstration video, it shows the user cloning an offer 5 times and then deleting a few that were not needed.  Having a popup at every delete would be too annoying, so the software checks if the removal is safe (i.e. does not incur a potential cost).  Conversely, an unsafe removal is one that would cause loss of paid fees and that warns the user with a popup.